doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md

```diff
@@ -5,4 +5,104 @@ permalink: /:path/
 sort_idx: 20
 ---
 
-SDK integration instructions for select platforms
+# The guide provides information on how to cross-compile C++ SDK for Raspberry PI
+
+**Verified against:**
+
+**Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
+
+**The further instructions must be executed on the host machine**
+
+1. Download and install toolchain.
+
+        mkdir rpi_root && cd rpi_root
+        export RPI_ROOT=$(pwd)
+        git clone https://github.com/raspberrypi/tools.git
+        export INSTALLDIR=$RPI_ROOT/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian
+        export PATH=$PATH:$INSTALLDIR/bin
+        export CROSS=arm-linux-gnueabihf
+        export CC=${CROSS}-gcc
+        export LD=${CROSS}-ld
+        export AS=${CROSS}-as
+        export AR=${CROSS}-ar
+
+2. Download and install zlib.
+
+        wget http://zlib.net/zlib-1.2.8.tar.gz
+        tar -xvzf zlib-1.2.8.tar.gz
+        cd zlib-1.2.8
+        ./configure --prefix=${INSTALLDIR}/arm-linux-gnueabihf/libc/usr
+        make && make install
+
+        cd $RPI_ROOT
+
+3. Download and install bzib2.
+
+        wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz
+        tar -xvzf bzip2-1.0.6.tar.gz
+        cd bzip2-1.0.6
+        sed -e "/^all:/s/ test//" Makefile > Makefile-libbz2_so
```

Comment:
Use `sed -e "/^all:/s/ test//" -i Makefile` to change the file in-place. You don't need `-f Makefile-libbz2_so` below.